.gitlab-ci.yml

```diff
@@ -357,3 +370,18 @@ staging-reset-scheduled-failure:
     - postmessage "#rum-deploy" "$MESSAGE_TEXT"
   dependencies:
     - staging-reset-scheduled
+
+merge-into-staging:
+  stage: pre-deploy
+  tags: ['runner:main', 'size:large']
+  image: $CI_IMAGE
+  only:
+    refs:
+      - main
+  except:
+    refs:
+      - schedules
+  before_script:
+    - eval $(ssh-agent -s)
+  script:
+    - node --no-warnings scripts/staging-ci/merge-into-staging.js
```

Comment:
Why do we need the `--no-warnings` flag?

Reply:
Good question, I got inspired by the check-licenses script. I'll try without